promoter/image/promoter.go

```diff
@@ -101,6 +102,12 @@ func (p *Promoter) PromoteImages(opts *options.Options) (err error) {
 		return errors.Wrap(err, "activating service accounts")
 	}
 
+	// Prewarm the TUF cache with the targets and keys. This is done
+	// to avoid collisions when signing and verifying in parallel
+	if err := p.impl.PrewarmTUFCache(); err != nil {
+		return errors.Wrap(err, "prewarming TUF cache")
```

Comment:
I'd say for new source code we probably should stick to:
```suggestion
		return fmt.Errorf("prewarming TUF cache: %w", err)
```

Reply:
OK, I've moved this while file to native error handling :+1: